base/ca/src/main/java/org/dogtagpki/server/ca/CAEngine.java

```diff
@@ -314,7 +315,7 @@ public CAPublisherProcessor getPublisherProcessor() {
     }
 
     public ICRLIssuingPoint getMasterCRLIssuingPoint() {
-        return crlIssuingPoints.get(CertificateAuthority.PROP_MASTER_CRL);
+        return crlIssuingPoints.get(ICertificateAuthority.PROP_MASTER_CRL);
```

Comment:
Actually I'm hoping that eventually we would be able to drop `ICertificateAuthority` and just use `CertificateAuthority`, but so far it's been difficult to drop `ICertificateAuthority` because of various dependencies. Could we move these constants to `CertificateAuthority` instead?

Reply:
I can do that. Was just taking a look at it to get a feel for the refactor and stopped even thinking about it when I saw `CertificateAuthority implements ICertificateAuthority, ICertAuthority`. That line enough is sufficient to tell me it will be complex :laughing:

Reply:
Actually even just moving the constants is sufficient to run into dependency issues so maybe I will just press ahead with the change as is.

Reply:
Before we drop `ICertificateAuthority`, I managed to drop `ICertAuthority` - so something went away at least :laughing: 
https://github.com/dogtagpki/pki/pull/3943